chapters/external-repository-identifiers.md

```diff
@@ -200,6 +239,17 @@ Examples:
 
 External documentation: [Software Heritage](https://docs.softwareheritage.org/devel/swh-model/persistent-identifiers.html)
 
+
+### F.4.1 gitbom <a name="gitbom"></a>
+
+Locator Format: gitoid":"<git object type>":"<hash algorithm>":"<hash value>
+	
+Contextual Example: gitoid:blob:sha1:261eeb9e9f8b2b4b0d119366dda99c6fd7d35c64
+
+External Documentation: <https://gitbom.dev/>
+
+Note: GitBOM uses a similar algorithm as Git to uniquely identify files by using their 40-byte hash. Gitoid stands for Git Object ID.  Git is an object store, see <https://git-scm.com/book/en/v2/Git-Internals-Git-Objects>.
```

Comment:
Formatting consistent with the other additions above

```suggestion
Documentation: GitBOM uses a similar algorithm as Git to uniquely identify files by using their 40-byte hash. `gitoid` stands for Git Object ID.  Git is an object store, see <https://git-scm.com/book/en/v2/Git-Internals-Git-Objects>.
```

Reply:
Looks like SWH section is not aligned with those above it therefore we initially made the GitBOM section. Propose I add a second commit to my PR to align the entire file to the same setup of fields per section.